HeterogeneousCore/CUDAUtilities/test/BuildFile.xml

```diff
@@ -0,0 +1,3 @@
+<bin   file="test_GPUSimpleVector.cu" name="test_GPUSimpleVector">
+  <flags   CUDA_FLAGS="-std=c++14"/>
```

Comment:
Do I guess correctly that this rule does not need `<use name="cuda"/>` because the `.cu` is compiled directly to the executable solely by `nvcc`?

Reply:
I think SCRAM automatically "uses" cuda for .cu files.
Would make sense to add it, though.

Reply:
And/or `<use name="HeterogeneousCore/CUDAUtilities"/>` .